src/index.ts

```diff
@@ -98,31 +98,34 @@ export class Faker {
   private loadDefinitions(): void {
     // TODO @Shinigami92 2022-01-11: Find a way to load this even more dynamically
     // In a way so that we don't accidentally miss a definition
-    Object.entries(DEFINITIONS).forEach(([t, v]) => {
-      if (typeof this.definitions[t] === 'undefined') {
-        this.definitions[t] = {};
+    Object.entries(DEFINITIONS).forEach(([moduleName, entryNames]) => {
```

Comment:
Maybe it's better and more readable when we just use a forof loop here

```suggestion
    for (const [t, v] of Object.entries(DEFINITIONS)) {
```

This suggestion will not work on it's own due to the closing brace